pkg/apis/datavolumecontroller/v1alpha1/types.go

```diff
@@ -101,6 +101,12 @@ const (
 	// ImportInProgress represents a data volume with a current phase of ImportInProgress
 	ImportInProgress DataVolumePhase = "ImportInProgress"
 
+	// CloneScheduled represents a data volume with a current phase of ImportScheduled
```

Comment:
CloneScheduled instead of ImportScheduled.

Reply:
Yeah you are right. See how easy it is to copy and paste and forget something. We should find a way to unify these into a single constant for all transfer type operations (import/clone/upload/whatever)